### ISSUE TITLE
Memory Leak in Digits 

### ISSUE THREAD

**Issue by njovy** [twitter/digits-android#9]
I am using [Leakcanary](https://github.com/square/leakcanary) to detect a memory leak. Whenever an activity which calls Digits.authenticate is being destroyed, Leakcanary detects a memory leak in com.digits.sdk.android.LoginResultReceiver.callback. Below is the copy of the logcat.
```
D/LeakCanary﹕ * GC ROOT android.os.ResultReceiver$MyResultReceiver.this$0
D/LeakCanary﹕ * references com.digits.sdk.android.LoginResultReceiver.callback
```

```
D/LeakCanary﹕ * Instance of android.os.ResultReceiver$MyResultReceiver
D/LeakCanary﹕ |   this$0 = com.digits.sdk.android.LoginResultReceiver [id=0x13220660]
D/LeakCanary﹕ |   mDescriptor = java.lang.String [id=0x7007e9e8]
D/LeakCanary﹕ |   mObject = -1212077312
D/LeakCanary﹕ |   mOwner = android.os.ResultReceiver$MyResultReceiver [id=0x13220840]
D/LeakCanary﹕ * Instance of com.digits.sdk.android.LoginResultReceiver
D/LeakCanary﹕ |   static $staticOverhead = byte[] [id=0x132b9541;length=24;size=40]
D/LeakCanary﹕ |   static KEY_ERROR = java.lang.String [id=0x13220640]
D/LeakCanary﹕ |   static RESULT_ERROR = 400
D/LeakCanary﹕ |   static RESULT_OK = 200
D/LeakCanary﹕ |   callback = com.oldroid.apps.votox.ui.signin.SigninActivity [id=0x12f09340]
D/LeakCanary﹕ |   sessionManager = com.twitter.sdk.android.core.PersistedSessionManager [id=0x12e781f0]
D/LeakCanary﹕ |   mHandler = null
D/LeakCanary﹕ |   mLocal = true
D/LeakCanary﹕ |   mReceiver = android.os.ResultReceiver$MyResultReceiver [id=0x13220840]
```

**Comment by rallat** [comment]
This issue has been fixed in the digits version 1.6.1
Please update your version and thank you for reporting it!

**Comment by njovy** [comment]
Awesome! Thank you very much!!
